src/server/db_slice.cc

```diff
@@ -457,8 +458,10 @@ DbSlice::ItAndExp DbSlice::FindInternal(const Context& cntx, std::string_view ke
   res.it = db.prime.Find(key);
   FiberAtomicGuard fg;
   if (!IsValid(res.it)) {
-    if (mode == FindInternalMode::kUpdateCacheStats)
+    if (mode == FindInternalMode::kUpdateReadStats)
       events_.misses++;
+    else
+      events_.mutations++;
```

Comment:
Nit: I'd use a `switch` here (and below), so that any changes to the `FindInternalMode` enum will be covered here (if we add a state, this will yield a compile warning unless specifically handled)